File: MyCode/JavaScript/src/Game.js
class Game {
	constructor (newWeek = 0,
								newHomeTeamRank = 0,
								newAwayTeamRank = 0,
								newDateTime = 'untimed'){
		this.week = newWeek
		this.homeTeamRank = newHomeTeamRank
		this.awayTeamRank = newAwayTeamRank
		this.dateTime = newDateTime
	}

	toString () {
    let result = `${this.week} ${this.homeTeamRank} ${this.awayTeamRank}  ${this.dateTime}`
    return result
	}
}

File: PreviousCode/Python/Game.py
class Game:
    def __init__(self, newWeek, newHomeTeamRank, newAwayTeamRank, newDateTime):
        self.week = newWeek
        self.homeTeamRank = newHomeTeamRank
        self.awayTeamRank = newAwayTeamRank
        self.dateTime = newDateTime

    def __str__(self):
        result = f"@ Week :{self.week},{self.dateTime}, Team {self.homeTeamRank} vs {self.awayTeamRank}"
        return result
